### PR TITLE
Add a proper README.

### PR DESCRIPTION
**Pull request by adam-thomas**
Clue's in the title.  This needed to happen :)

@incuna/backend review please?

### CODE REVIEW

**Comment by wytrych** [comment]
Looks really extensive! I like it

**Comment by adam-thomas** [comment]
Thanks :D

**Comment by adam-thomas** [comment]
@incuna/backend bump :)

File: README.md [hunk 1]
```diff
@@ -1,10 +1,71 @@
...
+An extensible Django app that provides forum functionality.
+- Administrators can create discussion groups.
+- Users can create discussions on groups, and comment on those discussions.
+- Users can also subscribe to groups and/or discussions to receive notifications from them, and post responses by replying to the notification emails. 
+
+## Installation
+
+`incuna-groups` is on PyPI, so you can install it with `pip install incuna-groups`.
+
+This project contains migrations, so run `python manage.py migrate` before using it.
```
Comment (KevinEtienne) [inline README.md]: do you need to state that you need to add it to `INSTALLED_APPS` to be able to run the migrations?

Comment (adam-thomas) [inline README.md]: oh yeah :)

Comment (adam-thomas) [inline README.md]: I also just found a sort-of API description for Mailgun :o

Comment (KevinEtienne) [inline README.md]: oh cool, are you going to add a reference here?

Comment (adam-thomas) [inline README.md]: yep, already did :)

Comment (adam-thomas) [inline README.md]: Hahah, what do you know - having found that API description, I got a file upload on the first try.  Shazam.

**Comment by adam-thomas** [comment]
Thanks :)

**Pull request merged**